server/startup/migrations/v171.js

```diff
@@ -0,0 +1,26 @@
+import {
+	Migrations,
+} from '../../../app/migrations';
+import {
+	Users,
+} from '../../../app/models';
+
+
+Migrations.add({
+	version: 171,
+	up() {
+		return Users.update(
```

Comment:
I think we are missing the Setting migration, we just removed the user preference, please check 'Accounts_Default_User_Preferences_roomCounterSidebar'